admin/tests/acceptance.rs

```diff
@@ -1,4 +1,4 @@
-#![deny(warnings, missing_docs, trivial_casts, unused_qualifications)]
+#![deny(rivial_casts, unused_qualifications)]
```

Comment:
```suggestion
#![deny(trivial_casts, unused_qualifications)]
```

We can probably also relax these to `warn`, but do `-D warnings` in CI

Reply:
Yeah, I agree we should get rid of `deny(warnings)` everywhere and use `RUSTFLAGS='-D warnings'`  on CI. That's orthogonal to this PR though - here I'm just relaxing the rules for rustsec-admin acceptance tests.